Runtime/ReactiveObjects/VertexFilters/VertexFilterByBoneComponent.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using JetBrains.Annotations;
+using UnityEngine;
+
+namespace nadena.dev.modular_avatar.core.vertex_filters
+{
+    [PublicAPI]
+    [AddComponentMenu("Modular Avatar/Vertex Filters/MA Vertex Filter - By Bone")]
+    [HelpURL("https://modular-avatar.nadena.dev/docs/reference/reaction/mesh-cutter?lang=auto")]
+    public class VertexFilterByBoneComponent : AvatarTagComponent, IVertexFilterBehavior
+    {
+        [SerializeField] internal Transform m_bone;
```

Comment:
Use `AvatarObjectReference` here, and implement `IHaveObjReferences`. Do not allow `Bone` itself to be directly set (callers will use `VFByBone.Bone.Set()` instead).